docs/ManualDeploymentGuideScalarDLAuditorOnAWS.md

```diff
@@ -64,23 +64,23 @@ so you need to add peering for internal communication between the Auditor, Ledge
     * Peering between Ledger and Client.
     * Peering between Auditor and Client.
 * Update route tables for VPC peering connection based on the [AWS official guide](https://docs.aws.amazon.com/vpc/latest/peering/vpc-peering-routing.html).
-* Add new Inbound and Outbound rule to restrict unwanted access to Ledger and Auditor.
-    * Add new Inbound rules to the Ledger Network ACLs to allow the Ledger Envoy LoadBalancer (e.g., 50051 and 50052 by default) access from the Auditor and Client.
+* Add new Inbound and Outbound rules to restrict unwanted access to Ledger and Auditor.
+    * Add new Inbound rules to Ledger Network ACLs to allow ephemeral ports (1024–65535) to access Ledger Envoy LoadBalancer (e.g., 50051 and 50052 by default) from the Auditor and Client.
         * You must set high priority to this rule.
-    * Add new Inbound rules to the Ledger Network ACLs to allow ssh access from the internet.
-        * You must set second priority to this rule.
-    * Add new Inbound rules to Ledger Network ACLs to allow all access to Ledger VPC.
-        * You must set third priority to this rule.
-    * Add new Outbound rules to the Ledger Network ACLs to allow all access to the internet.
-    * Add Ledger private subnets to subnet associations of the Ledger Network ACLs.
-    * Add new Inbound rules to the Auditor Network ACLs to allow the Auditor Envoy LoadBalancer (e.g., 40051, 40052 by default) access from the Ledger and Client.
+    * Add new Inbound rules to Ledger Network ACLs to restrict access to all ports except ephemeral ports from the Auditor and Client.
+        * You must set the next priority to this rule.
+    * Add new Inbound rules to Ledger Network ACLs to allow all access from the internet.
+        * You must set the next priority to this rule.
+    * Add new Outbound rules to Ledger Network ACLs to allow all access to the internet.
+    * Add all Ledger subnets to subnet associations of Ledger Network ACLs.
+    * Add new Inbound rules to Auditor Network ACLs to allow ephemeral ports (1024–65535) to access Auditor Envoy LoadBalancer (e.g., 40051 and 40052 by default) from the Ledger and Client.
         * You must set high priority to this rule.
```

Comment:
The above point is about adding rules to Auditor Network ACL. But this one says to set high priority to this rule.
If multiple rules have to be set, it will be confusing as to which rule should have high priority. 
Suggestion, change `rule` to `rules`.

Reply:
Thank you for your review comments.
Document updated with `rules` instead of `rule` 31dedd5